transactions/stakingCollection/register_node.cdc

```diff
@@ -8,22 +8,27 @@ transaction(id: String,
             networkingAddress: String,
             networkingKey: String,
             stakingKey: String,
-            amount: UFix64) {
+            amount: UFix64,
+            publicKeys: [[UInt8]]) {
```

Comment:
```suggestion
            machineAccountPublicKeys: [[UInt8]]) {
```

Reply:
Maybe this should be optional, since we don't create a machine account for all node roles?

Reply:
I was just going to have the `nil` case be an empty array but it is probably better and more explicit to just make it an optional. 👍